### PR TITLE
Template-Library/Local: Change - Allow manually print blank state.

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: includes/template-library/sources/local.php [hunk 1]
```diff
@@ -1293,11 +1293,31 @@ public function maybe_render_blank_state( $which ) {
...
 			$current_type = 'template';
-
-			$inline_style .= '#elementor-template-library-tabs-wrapper {display: none;}';
+			$additional_inline_style = '#elementor-template-library-tabs-wrapper {display: none;}';
 		}
 
+		$this->render_blank_state( $current_type, $additional_inline_style );
+	}
+
+	public function render_blank_state( $current_type, $additional_inline_style = '', $custom_description = '', $custom_href = '' ) {
```
Comment (matipojo) [inline includes/template-library/sources/local.php:1302]: Use one parameter `$args` and parse default args.

File: includes/template-library/sources/local.php [hunk 2]
```diff
@@ -1283,6 +1281,8 @@ public function maybe_render_blank_state( $which ) {
 			return;
 		}
 
+		$additional_inline_style = '';
```
Comment (matipojo) [inline includes/template-library/sources/local.php:1284]: Move the PR to `develop`.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: includes/template-library/sources/local.php (hunk 1 above)
Comment (iNewLegend) [reply includes/template-library/sources/local.php:1302]: Handled: https://github.com/elementor/elementor/pull/13037

File: includes/template-library/sources/local.php (hunk 2 above)
Comment (iNewLegend) [reply includes/template-library/sources/local.php:1284]: Handled: https://github.com/elementor/elementor/pull/13037

**Pull request closed without merging**